package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@signalk/tracks-plugin",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
I don't think any of the changes in this file are needed. Prettier maybe but it is already in module/package.json and there isn't much to prettify outside it.

Reply:
removed `prettier` from dependencies.